cunumeric/module.py

```diff
@@ -1392,6 +1414,214 @@ def squeeze(a: ndarray, axis: Optional[NdShapeLike] = None) -> ndarray:
     return a.squeeze(axis=axis)
 
 
+def broadcast_shapes(
+    *args: Union[NdShapeLike, Sequence[NdShapeLike]]
+) -> NdShape:
+    """
+
+    Broadcast the input shapes into a single shape.
+
+    Parameters
+    ----------
+    `*args` : tuples of ints, or ints
+        The shapes to be broadcast against each other.
+
+    Returns
+    -------
+    tuple : Broadcasted shape.
+
+    See Also
+    --------
+    numpy.broadcast_shapes
+
+    Availability
+    --------
+    Single CPU
+
+    """
+    return _broadcast_shapes(*args)
+
+
+def _broadcast_to(
+    arr: ndarray,
+    shape: NdShapeLike,
+    subok: bool = False,
+    broadcasted: bool = False,
+) -> ndarray:
+    # create an array object w/ options passed from 'broadcast' routines
+    arr = array(arr, copy=False, subok=subok)
```

Comment:
what's the point of this call? note that `subok` isn't handled in `array` anyway, so this is a no-op.

Reply:
broadcast has the argument and `numpy.array` in cuNumeric still has the argument even though it's not implemented for subclasses. This call is just aligned with the corresponding routines so that there's no need to change this call when subclasses are implemented.

Reply:
@magnatelee do you have a follow-up to the previous reply? I don't have enough context to know what is appropriate here (and below)